doc/files.md

```diff
@@ -761,21 +761,6 @@ URL embedLink = file.getPreviewLink();
 
 [get-preview-link]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxFile.html#getPreviewLink--
 
-Get Thumbnail
```

Comment:
Do you think we should leave this, but indicate that it is deprecated? Also, include how a snippet from your test of how to do it with the get representation api?

Reply:
@sujaygarlanka updated! get rep content is already documented (that's were I got the code snippet from 😄): https://github.com/box/box-java-sdk/blob/main/doc/files.md#get-representation-content